From ada11f58c54ba6d7c1f6252f404ae3fe45d62b66 Mon Sep 17 00:00:00 2001
From: David Abdurachmanov <david.abdurachmanov@sifive.com>
Date: Mon, 13 Sep 2021 03:15:35 -0700
Subject: [PATCH 6/7] riscv: sifive: unmatched: leave 128MiB for ramdisk

The current configuration only allows 125MiB, but the max allowed size should
be 128MiB.

Signed-off-by: David Abdurachmanov <david.abdurachmanov@sifive.com>
---
 include/configs/sifive-unmatched.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/sifive-unmatched.h b/include/configs/sifive-unmatched.h
index 3078b78e6b..d8eed92487 100644
--- a/include/configs/sifive-unmatched.h
+++ b/include/configs/sifive-unmatched.h
@@ -70,7 +70,7 @@
 	"scriptaddr=0x88100000\0" \
 	"pxefile_addr_r=0x88200000\0" \
 	"ramdisk_addr_r=0x88300000\0" \
-	"kernel_comp_addr_r=0x90000000\0" \
+	"kernel_comp_addr_r=0x90300000\0" \
 	"kernel_comp_size=0x4000000\0" \
 	"type_guid_gpt_loader1=" TYPE_GUID_LOADER1 "\0" \
 	"type_guid_gpt_loader2=" TYPE_GUID_LOADER2 "\0" \
-- 
2.27.0

